kiwi/command.py

```diff
@@ -26,7 +26,7 @@
 from builtins import bytes
 
 # project
-from .exceptions import KiwiCommandError
+from .exceptions import KiwiCommandError, KiwiCommandNotFound
```

Comment:
This should be in the form

```
from .exceptions import (
    KiwiCommandError,
    KiwiCommandNotFound
)
```

just to be consistent with the rest of the code